FSW/src/headers/timingClass.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef TIMINGCLASS_H
+#define TIMINGCLASS_H 
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+
+// NS2 headers
+
+/* - - - - - - Class Definitions - - - - - - */
+class ScienceMode

```

Comment:
Is there any reason to make this a class rather than just a static Mode enum? It seems simpler to do:
    static Mode scienceMode;
    scienceMode = STANDBY_MODE;
instead of:
    static ScienceMode scienceMode();
    scienceMode.set(STANDBY_MODE)